src/ResponsiveMasonry/index.js

```diff
@@ -49,35 +50,52 @@ const MasonryResponsive = ({
     750: 2,
     900: 3,
   },
+  gutterBreakPoints = {
+    350: DEFAULT_GUTTER,
+    750: DEFAULT_GUTTER,
+    900: DEFAULT_GUTTER,
```

Comment:
Not sure if we need default gutter breakpoints, since they're now defaulted ot `10px` already as can be seen in the `getResponsiveValue` function below.

Reply:
You are right, we don't need it. Can you remove it?